lib/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request.rb

```diff
@@ -34,7 +34,7 @@ class Request < ::AcaEntities::Operations::Transforms::Transform
                   map 'prior_insurance_end_date', 'Prior Insurance End Date'
                   map 'is_pregnant', 'Applicant Pregnant Indicator'
                   map 'children_expected_count', 'Number of Children Expected'
-                  map 'is_in_post_partum_period', 'Applicant Post Partum Period Indicator'
+                  map 'is_in_postpartum_period', 'Applicant Postpartum Period Indicator'
```

Comment:
_MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.